Lib/test/test_import/__init__.py

```diff
@@ -1339,6 +1342,39 @@ def test_circular_from_import(self):
             str(cm.exception),
         )
 
+    def test_unwritable_module(self):
+        package = inspect.cleandoc('''
```

Comment:
Any reason to not put this as a module in `test_import/data/`?

Reply:
Nope, I just didn't realize that was a thing :)